tests/performance/network/network_storage/network_storage.cpp

```diff
@@ -154,7 +156,7 @@ char                      *local_storage = nullptr;
 hpx::lcos::local::spinlock storage_mutex;
 
 //
-typedef struct {
+using test_options = struct test_options {
```

Comment:
This must be some old C code... It's enough to make it this: 
```suggestion
struct test_options {
```
It's just in C that `struct x` must be referred to as `struct x` unless there's a typedef.

Reply:
Aaah okay thanks!